### PR TITLE
[Product Partnerships] Added Timestream Quickstart & Updated Transcri…

### PR DESCRIPTION
**Pull request by rahul188**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/aws-transcribe/AsyncUserErrorCount.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Async User Error Count - Query
+name: Async User Error Count
```
Comment (aj-NR) [inline alert-policies/aws-transcribe/AsyncUserErrorCount.yml:1]: make the alert names as High AsyncUserErrorCount

File: alert-policies/aws-chatbot/MessageDeliveryFailure.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: High Message Delivery Failure - Query
+name: High Message Delivery Failure
```
Comment (aj-NR) [inline alert-policies/aws-chatbot/MessageDeliveryFailure.yml:1]: make the alert names as High MessageDeliveryFailure

File: alert-policies/aws-transcribe/SyncUserErrorCount.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Sync User Error Count - Query
+name: Sync User Error Count
```
Comment (aj-NR) [inline alert-policies/aws-transcribe/SyncUserErrorCount.yml:1]: same as above

File: alert-policies/aws-transcribe/AsyncServerErrorCount.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-name: Async Server Error Count - Query
+name: Async Server Error Count
```
Comment (aj-NR) [inline alert-policies/aws-transcribe/AsyncServerErrorCount.yml:1]: same as above

File: alert-policies/amazon-timestream/MagneticStoreRejectedUploadSystemFailures.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+name: MagneticStoreRejectedUploadSystemFailures
```
Comment (aj-NR) [inline alert-policies/amazon-timestream/MagneticStoreRejectedUploadSystemFailures.yml:1]: make name like High MagneticStoreRejectedUploadSystemFailures

File: alert-policies/amazon-timestream/MagneticStoreRejectedUploadUserFailures.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+name: MagneticStoreRejectedUploadUserFailures
```
Comment (aj-NR) [inline alert-policies/amazon-timestream/MagneticStoreRejectedUploadUserFailures.yml:1]: same as above

File: data-sources/amazon-timestream/config.yml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+id: amazon-timestream
+displayName: Amazon Timestream
+description: Monitor Amazon Timestream by sending Access logs to New Relic
```
Comment (aj-NR) [inline data-sources/amazon-timestream/config.yml:3]: what is accesslogs?

File: quickstarts/aws/amazon-timestream/config.yml [hunk 8]
```diff
@@ -0,0 +1,31 @@
+slug: amazon-timestream
+description: |-
+  ## What is Amazon Timestream?
+  Amazon Timestream is a fast, scalable, and serverless time-series database service that makes it easier to store and analyze trillions of events per day up to 1,000 times faster. Amazon Timestream automatically scales up or down to adjust capacity and performance, so that you don’t have to manage the underlying infrastructure.
+  ### Get started!
+  Start monitoring Amazon Timestream by using AWS Kinesis Firehose to send logs to New Relic! Take a look at our documentation for information on how to set this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-timestream-monitoring-integration/) to learn more about New Relic monitoring for Amazon Timestream.
+summary: |-
+  Monitor Amazon Timestream by streaming Firehose logs into New Relic
```
Comment (aj-NR) [inline quickstarts/aws/amazon-timestream/config.yml:10]: logs?

File: quickstarts/aws/amazon-timestream/config.yml [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+documentation:
+  - name: Amazon Timestream installation docs
+    description: |
+      Monitor Amazon Timestream by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-timestream-monitoring-integration/
+keywords:
+  - aws
+  - amazon web services
+  - networking
```
Comment (aj-NR) [inline quickstarts/aws/amazon-timestream/config.yml:25]: why networking will be keyword here?

File: alert-policies/aws-transcribe/AsyncUserErrorCount.yml (hunk 1 above)
Comment (rahul188) [reply alert-policies/aws-transcribe/AsyncUserErrorCount.yml]: Addressed, Updated Alert Name as  High AsyncUserErrorCount

File: alert-policies/aws-chatbot/MessageDeliveryFailure.yml (hunk 2 above)
Comment (rahul188) [reply alert-policies/aws-chatbot/MessageDeliveryFailure.yml]: Addressed, Updated Alert Name as  High MessageDeliveryFailure

File: quickstarts/aws/amazon-timestream/config.yml (hunk 9 above)
Comment (rahul188) [reply quickstarts/aws/amazon-timestream/config.yml]: Addressed, removed keyword networking & added timestream.

**Pull request merged**